modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -52,18 +52,30 @@ If you do not have access to Access Management, request access from your system
 
 == Generate a Certificate-Key Pair
 
-You’ll need a certificate-key pair for the internal load balancer in Runtime Fabric. If you have one already, both files must be of the .pem type, your keystore must have a passcode, and your Common Name should match the domain you’ll use in your Runtime Fabric. Otherwise, follow these steps to generate one for evaluation purposes.
+You’ll need a certificate-key pair for the internal load balancer in Runtime Fabric. If you have one already, both files must be of the .pem type, your keystore must have a passcode, and your Common Name should match the domain you’ll use in your Runtime Fabric. 
+
+RSA keys are the most common type of keys. RSA keys of 2K length offer the best compromise between security and performance. 
```

Comment:
I did not mark this earlier because I was not sure that 2K was 2 kilobytes. But now that I know...there should be a space between the 2 and the K.
GLOBAL; please search for 2K and fix. I won't mark each, to save time.

Reply:
```suggestion
RSA keys are the most common type of keys. RSA keys that are 2 kilobytes long offer the best compromise between security and performance. 
```